op-e2e/actions/interop/dsl/interop.go

```diff
@@ -123,6 +126,13 @@ func SetMessageExpiryTime(expiryTime uint64) setupOption {
 	}
 }
 
+func SetInteropOffset(offset uint64) setupOption {
+	return func(recipe *interopgen.InteropDevRecipe) {
+		fmt.Println("Setting interop offset in recipe:", offset)
```

Comment:
Was this for debugging? We usually don't just print to stdout for logging.

Reply:
Removed in c6fd3b743583a4bdbc626c27c8e5b1a8af210d24.